Game/assets/state_machine/SM3.stm

```diff
@@ -268,7 +268,7 @@ transitions:
   - transition_source: death
     transition_target: idle
     transition_trigger: isIdle
-    transition_blend: 300
+    transition_blend: 0
```

Comment:
Do we want this to be 0? I have no idea

Reply:
In this case, yes, we need no transition from death to idle.